provider/source/data/debug/time_zone/specific_long@1/en-001.json

```diff
@@ -1,576 +1,515 @@
 {
   "defaults": {
     "acre": {
-      "dt": "Acre Summer Time",
-      "st": "Acre Standard Time"
+      "-40": "Acre Standard Time"
     },
```

Comment:
Observation: you are losing some names. There is no "Acre Summer Time" in the new data file.

Reply:
Yes, because no time zone in that meta zone actually has a summer time. I think CLDR is inventing some names.

Reply:
Well it looks like you are only considering the tzdb footer, which has only the "latest" time zone data. It's possible that some of these zones have transitioned away from having a daylight savings transition, but had them previously.

Do I think we should include such data? Well, no, maybe not. It seems like a good way to cut out cold data, and worst cases is they just get the UTC offset printed.

Reply:
I think it makes sense to include only what is valid today. If we apply today's tz names to a date from 1980, people will understand. If we include tz names that haven't been used for 40 years, or worse, meant something different 40 years ago, that might be technically correct, but non tz-nerds will not interpret it correctly.